external/docker-integration-tests/pom.xml

```diff
@@ -130,15 +130,9 @@
       <artifactId>postgresql</artifactId>
       <scope>test</scope>
     </dependency>
-    <!-- Oracle ojdbc jar, used for oracle  integration suite for docker testing.
-     See https://github.com/apache/spark/pull/11306 for background on why we need
-     to use a an ojdbc jar for the testcase. The maven dependency here is commented
-     because currently the maven repository does not contain the ojdbc jar mentioned.
-     Once the jar is available in maven, this could be uncommented. -->
-    <dependency>
-      <groupId>com.oracle</groupId>
-      <artifactId>ojdbc6</artifactId>
-      <version>11.2.0.1.0</version>
+    <dependency>
+      <groupId>com.oracle.database.jdbc</groupId>
+      <artifactId>ojdbc8</artifactId>
```

Comment:
Hi, @gaborgsomogyi . 
`Support Oracle Kerberos login in JDBC connector` is a good contribution, but upgrading the existing `OracleIntegrationSuite` is also another independent good contribution.
Could you proceed upgrading `OracleIntegrationSuite` first as a separate JIRA?

Reply:
+1; this fix looks separate from this PR.

Reply:
Creating a jira + PR to separate it. Thanks.

Reply:
When the extract is done coming back to the rest of the comments and processing them...

Reply:
Created SPARK-32049